pkgs/data/documentation/nixpkgs-manual/lib-function-locations.nix

```diff
@@ -2,80 +2,100 @@
   lib,
   path,
   writeText,
-  nixpkgs ? { },
-  libsets
+  nixpkgs,
+  libsets,
 }:
 let
+  inherit (lib)
+    attrNames
+    concatMap
+    concatStringsSep
+    filter
+    flatten
+    isAttrs
+    length
+    listToAttrs
+    map
```

Comment:
map is just available without builtins or lib, we don't need to take it from lib.